ur_robot_driver/src/ros/hardware_interface.cpp

```diff
@@ -695,10 +695,14 @@ void HardwareInterface::extractRobotStatus()
   {
     robot_status_resource_.motion_possible = TriState::FALSE;
   }
-  else
+  else if (robot_mode_ == ur_dashboard_msgs::RobotMode::RUNNING || robot_mode_ == ur_dashboard_msgs::RobotMode::BACKDRIVE)
```

Comment:
Can the driver command motion if the robot is in `BACKDRIVE` mode?

Reply:
technically, I think it could. By putting the robot into freedrive mode, it can be moved physically. Putting the robot into freedrive mode would require sending script code to the robot manually.

@gavanderhoorn: Would you suggest to remove that edge case and only use `RobotMode::RUNNING`?

Reply:
I would indeed remove it.

`motion_possible` should reflect whether it is possible to command the robot from the ROS driver/component "as-is" (ie: in the state it currently is in).

That would not include hypotheticals I believe.